src/applications/auth/containers/AuthApp.jsx

```diff
@@ -62,13 +63,23 @@ export default function AuthApp({ location }) {
     setHasError(true);
   };
 
+  const { useToggleValue, TOGGLE_NAMES } = useFeatureToggle();
+  const isInterstital = useToggleValue(TOGGLE_NAMES.mhvInterstitialEnabled);
+
   const redirect = () => {
     // remove from session storage
     sessionStorage.removeItem(AUTHN_SETTINGS.RETURN_URL);
 
     // redirect to my-va if necessary
     const updatedUrl = generateReturnURL(returnUrl);
 
+    if (
+      isInterstital &&
+      (loginType === 'mhv' || loginType === 'myhealthevet')
+    ) {
+      window.location.replace('/sign-in-changes-reminder');
```

Comment:
Let's make sure we re-save the `authReturnUrl` in session storage before we redirect to the `sign-in-changes-reminder`. Also let me ask UX + Joe if we should be doing this for all authentications (USiP included).